lib/config/generator.js

```diff
@@ -0,0 +1,210 @@
+var path = require('path');
+var Vow = require('vow');
+var vowNode = require('vow-node');
+var Table = require('cli-table');
+var prompt = require('prompt');
+var fs = require('fs');
+
+var Checker = require('../checker');
+var utils = require('../utils');
+
+prompt.message = '';
+prompt.delimiter = '';
+prompt.start();
+
+/**
+ * Script that walks the user through the autoconfig flow
+ * @type {String[]}
+ */
+var prompts = [
+    {
+        name: 'Please choose a preset number:'.green,
+        require: true,
+        pattern: /[0-9]*/
```

Comment:
> You mean from the plugin?

Yup. I'm not sure if we can look in `node_modules` or `.env.NODE_PATH` for plugins (how to differentiate them from others, by `jscs-` prefix?), but it would be great if we could load `.jscsrc` and read `plugins` or `preset` options, or just provide `--plugins` option to cli to set them explicitly.

Another idea is to have `~/.jscsrc` file and use it as default (with plugins, presets, etc.).